x-pack/plugins/xpack_main/public/views/management/telemetry/manage_telemetry_page.less

```diff
@@ -0,0 +1,4 @@
+.manageTelemetryPage {
+  background-color: #f5f5f5;
+  min-height: 100vh;
```

Comment:
Can you set the min-height to: `min-height: calc(~"100vh - 70px");` This will incorporate the management tabs.